code/game/objects/structures/stool_bed_chair_nest/stools.dm

```diff
@@ -4,6 +4,8 @@
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "stool"
 	anchored = 1.0
+	var parts = 1
+	var is_wooden = 0

```

Comment:
This is a start, but still not the greatest. `parts` isn't a particularly good var name here; someone is probably going to have to look up what it does. Also, `is_wooden` isn't a good var here, either; there's more than just wooden and metal chairs.

get rid of these, and replace them with:

```
	var/buildstacktype = /obj/item/stack/sheet/metal
	var/buildstackamount = 2
```

Then adjust the `buildstackamount` by however many sheets something drops.

You'd also use `buildstacktype` where it's deconned by animals.

also, don't use `var parts`; use `var/parts`.